blazesym-go/symbolizer.go

```diff
@@ -0,0 +1,121 @@
+package blazesym
+
+import (
+	"unsafe"
+)
+
+/*
+#cgo LDFLAGS: -lblazesym_c
+#include "blazesym.h"
+
+// Adding a C function to return syms from blaze_result
+struct blaze_sym* get_result(blaze_syms* res, size_t pos) {
+	return &res->syms[pos];
+}
+*/
+import "C"
+
+// Symbolizer represents a Blazesym symbolizer.
+type Symbolizer struct {
+	s *C.blaze_symbolizer
+}
+
+// NewSymbolizer creates an instance of a symbolizer.
+// See: https://docs.rs/blazesym-c/latest/blazesym_c/fn.blaze_symbolizer_new.html
+func NewSymbolizer() (*Symbolizer, error) {
+	s := C.blaze_symbolizer_new()
+	if s == nil {
+		return nil, BlazeErr(C.blaze_err_last()).Error()
+	}
+
+	return &Symbolizer{s: s}, nil
+}
+
+// Close closes the a symbolizer.
+func (s *Symbolizer) Close() {
+	C.blaze_symbolizer_free(s.s)
+}
+
+// SymbolizeElfVirtOffsets symbolizes virtual offsets in an ELF file.
+// See: https://docs.rs/blazesym-c/latest/blazesym_c/fn.blaze_symbolize_elf_virt_offsets.html
+func (s *Symbolizer) SymbolizeElfVirtOffsets(path string, debugSyms bool, stack []uint64) ([]Sym, error) {
```

Comment:
I find the "stack" terminology confusing. There is no requirement for anything to resemble or origingate from a stack. Can we just go with `addrs` as the main library does?